contracts/solidity/inhetitance/Main.sol

```diff
@@ -14,8 +14,4 @@ contract Main is Base {
     function returnSuper() public view virtual returns (string memory) {
         return super.classIdentifier();
     }
-
-    function destroyContract(address recipient) public {
```

Comment:
Please restore.
We can remove it separately but until Cancun let's maintain the coverage